ballista/executor/src/executor_process.rs

```diff
@@ -365,7 +366,7 @@ pub async fn start_executor_process(opt: ExecutorProcessConfig) -> Result<()> {
             })
             .await
         {
-            error!("error sending heartbeat with fenced status: {:?}", error);
+            error!(executor_id, error = %error, "error sending heartbeat with fenced status");
```

Comment:
tip: can be written as
```suggestion
            error!(executor_id, %error, "error sending heartbeat with fenced status");
```